mmf/utils/download.py

```diff
@@ -423,6 +423,13 @@ def move(path1, path2):
     shutil.move(path1, path2)
 
 
+def get_copy(path1, path2):
```

Comment:
Python has a `copy` utility so I thought it would be best to use a different name for the method to avoid any future naming clash.
Do you think I should still rename to copy?

Reply:
Yeah, namespacing will make sure you don't import wrong one.

Reply:
So I can leave it it as `get_copy()` ?

Reply:
No, change it to `copy` as namespacing will make sure you get correct one.